executor/common_linux.h

```diff
@@ -2514,8 +2514,12 @@ static long syz_socket_connect_nvme_tcp()
 		return -1;
 	int sock = syscall(__NR_socket, AF_INET, SOCK_STREAM, 0x0);
 	int err = errno;
-	if (setns(netns, 0))
-		fail("setns(netns) failed");
+	if (setns(netns, 0)) {
+		// The operation may fail if the fd is closed by
+		// a syscall from another thread.
+		exitf("setns(netns) failed");
+		return -1;
```

Comment:
exitf should be declared noreturn, no need for explicit return after it.

Reply:
Done